src/scss/_variables.scss

```diff
@@ -0,0 +1 @@
+$blue:    #bb5151;
```

Comment:
Can't seem to find where you are using this to prove you successfully override this

Reply:
Just use this variable in your styling